internal/executor/vaultunboxer/boxedvalue.go

```diff
@@ -57,17 +61,36 @@ func NewBoxedValue(rawBoxedValue string) (*BoxedValue, error) {
 		}
 	}
 
+	vaultPathArgs := map[string][]string{}
+	if len(parts) > 2 {
+		for _, arg := range parts[2:] {
+			argParts := strings.Split(arg, "=")
+			if len(argParts) != 2 {
+				return nil, fmt.Errorf("%w: found an invalid argument '%q'", ErrInvalidBoxedValue, arg)
+			}
+			if argParts[0] == "" || argParts[1] == "" {
+				return nil, fmt.Errorf("%w: found an unvalid argument '%q'", ErrInvalidBoxedValue, arg)
```

Comment:
`%q` already quotes the value and also let's explain why the argument is invalid:

```suggestion
				return nil, fmt.Errorf("%w: found an invalid argument %q: key and/or value are empty", ErrInvalidBoxedValue, arg)
```